### PR TITLE
update hackathon page

### PR DESCRIPTION
**Pull request by Remi-Gau**
just added a quick modification with at the top of the hackathon page.

nothing big.

tempted to add an image for the hubs...

### CODE REVIEW

**Comment by cmaumet** [comment]
@Remi-Gau: Thanks for this! Can you doublecheck if [push from maintainers is enabled](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests)? I've just made a few updates but push is failing for some reason...

**Comment by Remi-Gau** [comment]
Seems so...

Or am I missing something?

Screenshot 2020-05-06 17:23:42

**Comment by cmaumet** [comment]
Okay I need to get back to my git skills... Sorry for the troubles! Let me try again!

**Comment by cmaumet** [comment]
That was an issue in the naming of my local branch (was `remi/remi-virtual_hackathon_announce` and now works when updated to `remi-virtual_hackathon_announce`).

Can you check you are okay with my edits?

**Comment by Remi-Gau** [comment]
Good with me.

Feel free to merge. :-)

**Comment by cmaumet** [comment]
Great! Let's merge (but I'll wait to publish until we have all our comm channels ready)
